src/lnp/session/noise/act.rs

```diff
@@ -0,0 +1,178 @@
+// LNP/BP Core Library implementing LNPBP specifications & standards
+// Written in 2020 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
```

Comment:
I think the credits should go here, instead of my name. We need to place (1) authors of Rust lightning original handshake code (Matt Corallo/Antonie Riard), (2) guys that proposed PR and youself. So probably:

```suggestion
//     Julian Knutsen
//     Rajarshi Maitra
//     Arik Sosman
//     Matt Corallo
//     Antoine Riard
```

Reply:
Done